index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="app-wrap">
+        <header class="app-header">
+            <div class="logo"><p class="text_logo">My Logo</p></div>
+            <nav class="nav-bar">
+                 <div class="nav">
+                    <img class="menu" src="Union.png" alt="menu">
+                </div>  
+            </nav>
+        </header>
+        <div class="content">
+            <h2 class="salute">I'm Alexandra. Glad to see you!</h2>

```

Comment:
- [ ] Kindly use `h1` tag for the main title of your page for the best practices of HTML & CSS. 👍🏽